File: crates/semantiq-parser/tests/fixtures/python/sample.py
import os
from collections import OrderedDict

class User:
    def __init__(self, name):
        self.name = name

    @staticmethod
    def from_dict(data):
        return User(data["name"])

    def greet(self):
        return f"Hello, {self.name}"

def process(items):
    return items

File: crates/semantiq-parser/tests/fixtures/go/sample.go
package main

import "fmt"

type User struct {
	Name string
}

type Greeter interface {
	Greet() string
}

func (u *User) Greet() string {
	return fmt.Sprintf("Hi %s", u.Name)
}

func main() {
	fmt.Println("hi")
}

const Pi = 3.14
var counter = 0
